src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/DocumentHelperTest.java

```diff
@@ -233,11 +308,14 @@ public void buildDocumentContent2() {
                 + "\"Case_No\":\"123456\",\n"
                 + "}\n"
                 + "}\n";
+        caseDetails2.getCaseData().getRespondentCollection().get(0).getValue().setResponseStruckOut(NO);
+        caseDetails2.getCaseData().getRepCollection().get(0).getValue().setRespRepName("Antonio Vazquez");
         assertEquals(expected, DocumentHelper.buildDocumentContent(caseDetails2.getCaseData(), "",
                 userDetails, MANCHESTER_CASE_TYPE_ID, venueAddressInputStream,
                 caseDetails2.getCaseData().getCorrespondenceType(),
                 caseDetails2.getCaseData().getCorrespondenceScotType(), null,
                 null).toString());
+        caseDetails2.getCaseData().getRepCollection().get(0).getValue().setRespRepName("RepresentativeNameRespondent");
```

Comment:
Does this line serve any purpose?

Reply:
Unfortunately other tests fail, if this line is not there. We have to reset the RespRepName to the initial value as it was in CaseDetailsTest2.json.